maps/destiny.dmm

```diff
@@ -42583,12 +42584,13 @@
 /area/station/crew_quarters/arcade)
 "oEE" = (
 /obj/machinery/door/firedoor/border_only,
-/obj/machinery/door/airlock/gannets{
-	name = "airlock-'Crew Lounge'"
-	},
 /obj/cable{
 	icon_state = "4-8"
 	},
+/obj/machinery/door/airlock/gannets{
+	name = "airlock-'Janitor Supply Closet";
+	req_access_txt = "26"
```

Comment:
Sanity check on setting this value vs. using an access spawner?

Reply:
so, almost none of destiny actually uses access spawners. there's a single one on an airlock into science that has had some issues with maint/sci shared access stuff as a part of an earlier bug fix. i'm not looking to convert destiny to access spawners at this time and intend to do that down road; preferably also around the time or as a part of a few prs for when/if i go in to replace all the doors for perspective ones.

Reply:
Not going to verify as I don't do mapping, but I'm fine with consistency within the map. If the rest of this map uses var setting then I'm okay with it here.